Userland/Libraries/LibGfx/ImageFormats/CCITTDecoder.cpp

```diff
@@ -420,6 +420,19 @@ struct CCITTStatus {
     bool has_reached_eol { false };
 };
 
+ErrorOr<void> ensure_invalid_result_is_actually_eol(BigEndianInputBitStream& input_bit_stream, InvalidResult partially_read_eol)
```

Comment:
nit: I'd spell out "NFC" in the commit description. (I usually write "No behavior change" instead of "No functional change", but eh.)